src/Pages/Dom-module/domPage.js

```diff
@@ -0,0 +1,21 @@
+const city = document.querySelector(".city");
+const Temp = document.querySelector(".temp");
```

Comment:
some of the variables are starting from lowercase and some in Uppercase. Why?

Reply:
I have used Uppercase for differentiate between imported variables and object that i created from destructuring assignment since javascript is case sensitive